distribution/src/config/log4j2.properties

```diff
@@ -7,14 +7,37 @@ logger.action.level = debug
 appender.console.type = Console
 appender.console.name = console
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = [%d{ISO8601}][%-5p][%-25c{1.}] [%node_name]%marker %m%n
+appender.console.layout.pattern ={\
```

Comment:
as per comment on the [issue](https://github.com/elastic/elasticsearch/issues/32850#issuecomment-448661944) should we allow users to change that pattern? If so, how do we validate that the JSON format is not broken

Reply:
That was solved by creating ESJsonLayout which users won't be able to change